### PR TITLE
Ggm/clean up conf test

### PR DESCRIPTION
**Pull request by gabegma**
## Description:
- Clean-up fixtures and and various tests.
- Fix bug where `NO_SMART_TAGS` and `NO_DATA_ACTIONS` were added to the dataset as columns, but the values were never the right ones. This was due because `is not` in  `[a for a in ALL_DATA_ACTION_FILTERS if a is not DataAction.no_action]`  was not working. `!=` works.

## Checklist:

You should check all boxes before the PR is ready. If a box does not apply, check it to acknowledge
it.

* [x] **PRE-COMMIT.** You ran pre-commit on all commits, or else, you
  ran `pre-commit run --all-files` at the end.
* [x] **FRONTEND TYPES.** Regenerate the front-ent types if you played with types and routes.
  Run `cd webapp && yarn types` while the back-end is running.
* [x] **USER CHANGES.** The changes are added to CHANGELOG.md and the documentation, if they impact
  our users.
* [x] **DEV CHANGES.**
    * Update the documentation if this PR changes how to develop/launch on the app.
    * Update the `README` files and our wiki for any big design decisions, if relevant.
    * Add unit tests, docstrings, typing and comments for complex sections.

### CODE REVIEW

File: tests/utils.py [hunk 1]
```diff
@@ -1,9 +1,101 @@
...
+)
+
+# Sentiment Analysis
+PIPELINE_CFG = {
+    "name": "Default Model",
+    "model": {
+        "class_name": "tests.test_loading_resources.load_hf_text_classif_pipeline",
+        "kwargs": {"checkpoint_path": _CHECKPOINT_PATH},
+    },
+    "postprocessors": [{"threshold": 0.7}, {"temperature": 3}],
```
Comment (Dref360) [inline tests/utils.py:47]: threshold before temp?

**Review by Dref360 — APPROVED** [review]
Great stuff!
Minor comment

**Comment by gabegma** [comment]
I found a bug while making changes to fix the tests. I added it in the description of the PR!

**Comment by Dref360** [comment]
Ah great!

LGTM

**Pull request merged**